Cargo.toml

```diff
@@ -69,11 +69,15 @@ version = "0.1"
 [dependencies.tar]
 version = "0.4"
 
+[dependencies.which]
+version = "5.0.0"
```

Comment:
`which` is at v6.0.0 currently. any particular reason we are pinning to v5.0.0.

Reply:
https://crates.io/crates/which/6.0.0

Reply:
Also perhaps for another PR, but maybe we can use this to replace our `whereis` implementation? https://github.com/balena-os/takeover/blob/fe2581c560b6ed9b12c4a14cde3a02b451f444dc/src/common.rs#L77

Reply:
Ah, good catch! At some point I downgraded it because of compatibility issues with the toolchain version we were using at the time. I just pushed an update using 6.0.0.